Tasks/Common/docker-common/dockercommandutils.ts

```diff
@@ -89,6 +91,38 @@ export async function getLayers(connection: ContainerConnection, imageId: string
     return layers.reverse();
 }
 
+export function getImageSize(layers: { [key: string]: string }[]): string {
+    let imageSize = 0;
+    for (const layer of layers) {
+        for (let key in layer) {
+            if (key.toLowerCase() === "size") {
+                const layerSize = extractSizeInBytes(layer[key]);
+                imageSize += layerSize;
+            }
+        }
+    }
+
+    return imageSize.toString() + "B";
```

Comment:
should we use the formula to show B, KB or M kind of thing?

Reply:
Created bug on myself for handling the pending review comments. https://dev.azure.com/mseng/AzureDevOps/_workitems/edit/1529319